config/locales/en.yml

```diff
@@ -286,6 +286,7 @@ en:
       edit_application: "Edit Application"
       delete_application: "Delete Application"
       update: "Update"
+      please_select: "FOO"
```

Comment:
This was us just messing around...we should make sure to remove this if it's not working.